README.md

```diff
@@ -2,36 +2,82 @@
 
 ![][status]
 
-The [HACL*] repository is a collection of high-assurance cryptographic algorithms developed as part of [Project Everest].
-It includes source code written in [F*], generated code in C, verified assembly code
-from the [Vale] project, and an agile multiplexed cryptographic provider called [EverCrypt].
-As such, the full [HACL*] repository contains many software artifacts and a complicated build system
-that can be intimidating to a crypto developer who simply wishes to use verified crypto.
-
-This repository addresses this gap by presenting several usable crypto packages developed by Cryspen on top of HACL\*.
+This repository contains ready-to-use crypto packages developed by [Cryspen] on top of [HACL*].
 In particular, it contains a portable C crypto library that selects optimized implementations for each platform,
-as well as Rust, OCaml, and JavaScript bindings for this library. Cryspen is in the process of adding more usable APIs for crypto
-primitives, as well as extensive documentation for these APIs. Cryspen is also working on more optimized versions of some
-algorithms.
+as well as Rust, OCaml, and JavaScript bindings for this library.
+
+Currently, we are in the process of adding more usable APIs, [providing extensive documentation](https://tech.cryspen.com/hacl-packages/), and further optimizing the algorithms.
+
+## What is HACL\* and why should you use HACL Packages?
 
-## Build
+[HACL*] is a collection of high-assurance cryptographic algorithms developed as part of [Project Everest].
+It includes source code written in [F*], generated C code, verified assembly code from the [Vale] project, and an agile multiplexed cryptographic provider called [EverCrypt].
+As such, the full HACL\* repository contains many software artifacts and a complicated build system that can be intimidating to a crypto developer who simply wishes to use verified crypto.
 
-We uses [cmake] to configure the C build and [ninja] to build.
+## Quickstart
 
-Quick start: `./mach build --test`
+### Install build dependencies
 
-Build dependencies
+We need the following dependencies ...
 
 - [cmake] (3.17 or newer)
 - [ninja] (1.10 or newer)
 - [python] (3.6 or newer)
+- [clang] (X.X or newer) or [gcc] (X.X or newer)
```

Comment:
GCC 7 or Clang 7 is what we currently test. That would be good minimum versions for now.

Reply:
Resolved in b1c8928ea6eeab1f0d8b18fa7697b2e167df0968. (I'll squash later.)